integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -29,14 +29,16 @@ const parseSvg = (svgUrl: string): string => atob(svgUrl.replace("data:image/svg
 
 const insertIcon = (editor: Editor, tinymceConfig: HTMLElement, name: string): void => {
     const icon = tinymceConfig.getAttribute(`data-${name}-icon-src`);
-    editor.insertContent(`<img src="${icon}" style="width:15px; height:15px">`);
+    const alt = tinymceConfig.getAttribute(`data-${name}-icon-alt`);
+    editor.insertContent(`<img src="${icon}" alt="${alt}" style="width:15px; height:15px">`);
 };
 /* This function adds an icon which can be inserted in the content */
-const addIcon = (editor: Editor, tinymceConfig: HTMLElement, name: string, shortcut: string): void => {
+const addIcon = (editor: Editor, tinymceConfig: HTMLElement, name: string, alt: string, shortcut: string): void => {
     /* eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require */
     editor.ui.registry.addIcon(name, parseSvg(require(`../../svg/${name}.svg`)));
     editor.ui.registry.addMenuItem(name, {
         text: tinymceConfig.getAttribute(`data-${name}-icon-text`),
+        alt: tinymceConfig.getAttribute(`data-${name}-icon-alt`),
```

Comment:
```suggestion
```
Webpack can't run through with this line as it is not defined. Just removing it worked for me though :) I'm not sure if I'm not thinking of something :)